tests/test_entities.py

```diff
@@ -3303,14 +3288,12 @@ def test_upload_content_v2(self):
 
         """
         kwargs = {'kwarg': gen_integer()}
-        with mock.patch.object(client, 'post') as post:
-            with mock.patch.object(
-                entities,
-                '_handle_response',
-                return_value={'status': 'failure'},
-            ) as handler:
-                with self.assertRaises(entities.APIResponseError):
-                    self.repo.upload_content(**kwargs)
+        with mock.patch.object(client, 'post') as post, mock.patch.object(
```

Comment:
The nested looks more cleaner here than combined! Not a blocker though.

Reply:
I also agree in some instances, most of which I didn't allow formatting on, but others it works better. this one is a bit on the edge, but we can keep it for now since reverting wouldn't necessarily be worth the effort.